test/integration/targets/openssh_keypair/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+- name: Generate privatekey1 - standard
+  connection: local
+  openssh_keypair:
+    path: '{{ output_dir }}/privatekey1'
+
```

Comment:
I think you should also include at least one idempotency check for `state == present` and one for `state == absent`. Right now you're not testing the check code at all.

Reply:
The same issue exists for the other modules in the `crypto` namespace

e.g.

https://github.com/ansible/ansible/blob/03bccdda9223f8b6ab4bb38a264e07b67f4ed890/test/integration/targets/openssl_privatekey/tasks/main.yml#L15-L22

and the check:

https://github.com/ansible/ansible/blob/03bccdda9223f8b6ab4bb38a264e07b67f4ed890/test/integration/targets/openssl_privatekey/tests/validate.yml#L31-L39

which doesn't check for idempotency of the absent state. I guess a followup PR could be made to fix all modules in the namespace at once (e.g. first checking for file existence and only reporting "changed" if they existed).

Reply:
Yep, that's something we should change for all tests. Let's do that once this is merged.